rootdir/root/init.qcom.rc

```diff
@@ -460,7 +424,6 @@ service qmuxd /system/bin/qmuxd
 
 service netmgrd /system/bin/netmgrd
     class main
-    disabled
```

Comment:
I think I found it. Android cannot find getprop. Adding PATH=/system/bin:$PATH to init.class_main.sh makes it work.